website/project/model.py

```diff
@@ -2119,7 +2119,11 @@ def fork_node(self, auth, title='Fork of '):
                 if forked_node is not None:
                     forked.nodes.append(forked_node)
 
-        forked.title = title + forked.title
+        if title == 'Fork of ' or title == '':
```

Comment:
Okay.  Will be a part of https://github.com/CenterForOpenScience/osf.io/pull/5611